package.json

```diff
@@ -23,7 +23,7 @@
   "devDependencies": {
     "backbone": "1.2.1",
     "chai": "^3.0.0",
-    "karma": "^0.12.36",
+    "karma": "^0.13.9",
```

Comment:
I removed the sinon change and the tests still pass on travis.  Sorry for all the noise on this small PR.